### DISCUSSION TITLE
Please help me with autostart not working

### DISCUSSION THREAD

**Discussion by joseph-ui-designer** [FundamentalFrequency/.github#121] [Q&A]
I followed the steps in the Bulid Guide, but the autostart is not working.
I'm using a Raspberry Pi 4 and the OS is using the 04/04/2022 release version (Bullseye 64bit).
Here's the process I went through

<1. Modifications to **LMN_start.sh** documentation>

#!/bin/bash
sleep 2
/home/pi/LMN-3

$ chmod +x ~/LMN_start.sh

<2. Modifications to **LMN-3.desktop** documentation (After creating the autostart directory)>

[Desktop Entry]
Type=Application
Name=LMN-3
Exec=/home/pi/LMN_start.sh

<3. Reboot>

If you have any issues with the process I've followed, please help 🙏🙏🙏
If there's something wrong with my process, is there a way to bypass it?

**Comment by vidalsasun** [comment]
**this is my LMN_start.sh (Sorry for the log in spanish). with this script LMN3 runs always in my RPI 4 8GB:** 

```
#!/bin/bash
echo "$(date) - Iniciando script LMN_start.sh" >> /home/pi/restart.log
# Detener el proceso LMN-3 si está en ejecución
pgrep LMN-3 | xargs kill -9
echo "$(date) - Cerrando LMN-3" >> /home/pi/restart.log
# Verificar si el ejecutable de LMN-3 existe y es ejecutable
if [ -x "/home/pi/LMN-3" ]; then
    echo "$(date) - /home/pi/LMN-3 existe y es ejecutable. Iniciando..." >> /home/pi/restart.log
    /home/pi/LMN-3 &
    echo "$(date) - Nueva instancia de LMN-3 iniciada." >> /home/pi/restart.log
else
    echo "$(date) - Error: /home/pi/LMN-3 no es ejecutable o no existe." >> /home/pi/restart.log
fi
echo "$(date) - Script LMN_start.sh completado" >> /home/pi/restart.log
```

**and this is my /home/pi/.config/autostart/LMN-3.desktop:**

[Desktop Entry]
Type=Application
Name=LMN-3
Exec=/home/pi/LMN_start.sh

**Reply by HotPTgz** [reply]
Hey there, you may already have found a solution by now. I built one the other day and I had the same problem... until I realized that my username is not "pi". Once I replaced that, it worked like a charm.

Something like this:

#!/bin/bash
sleep 2
_/home/your user name goes here/LMN-3_

**Reply by HotPTgz** [reply]
And same for the other modification:

[Desktop Entry]
Type=Application
Name=LMN-3
_Exec=/home/your user name goes here/LMN_start.sh_

So for example, if your user name is Potato, the correct path would be /home/Potato/LMN-3
